contrib/gzstream/Makefile.old

```diff
@@ -1,107 +0,0 @@
-# ============================================================================
-# gzstream, C++ iostream classes wrapping the zlib compression library.
-# Copyright (C) 2001  Deepak Bandyopadhyay, Lutz Kettner
-# 
```

Comment:
Is their license + copyright info elsewhere in the package too?  We don't want to strip that out entirely.